domain-model/src/test/java/uk/gov/hmcts/cmc/domain/models/ResponseTest.java

```diff
@@ -27,10 +29,46 @@ public void shouldHaveNoValidationMessagesWhenResponseDataIsValid() {
         assertThat(response).isEmpty();
     }
 
+    @Test
```

Comment:
We could call it `...WhenPartAdmissionDataIs...`. Here and in other tests. Would be nice to say the same for full defence tests.